client/knode/api.go

```diff
@@ -179,25 +179,22 @@ type TransferArgs struct {
 // PublicTokenAPI exposes a collection of methods related to tokens
 type PublicTokenAPI struct {
 	accountMgr *accounts.Manager
-	token      token.Token
+	consensus  consensus.Consensus
 }
 
-func NewPublicTokenAPI(accountMgr *accounts.Manager, token token.Token) *PublicTokenAPI {
+func NewPublicTokenAPI(accountMgr *accounts.Manager, token consensus.Consensus) *PublicTokenAPI {
```

Comment:
`token consensus.Consensus` seems wrong. Maybe `cons` or even `c`?

Reply:
`c` is my choice!